uaclient/config.py

```diff
@@ -368,7 +368,7 @@ def _remove_beta_resources(self, response) -> "Dict[str, Any]":
                 released_resources.append(resource)
                 continue
 
-            if not ent_cls.is_beta:
+            if not ent_cls.is_beta or resource.get("status", "") == "enabled":
```

Comment:
I think intead of a static string here we want to use our defined UserFacingStatus enum
```suggestion
            if not ent_cls.is_beta or resource.get("status", "") == UserFacingStatus.ACTIVE.value
```